yogi2/src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -53,8 +53,8 @@ public class DriveTrain extends SubsystemBase implements Sendable
   private MotorControllerGroup left = new MotorControllerGroup(leftFront, leftRear);
   private MotorControllerGroup right = new MotorControllerGroup(rightFront, rightRear);
   private DifferentialDrive forwardDrive = new DifferentialDrive(left, right);
-  private DifferentialDrive reverseDrive = new DifferentialDrive(right, left);
-
+//  private DifferentialDrive reverseDrive = new DifferentialDrive(right, left);
```

Comment:
put a comment about why this is commented out (and for lines below)

Reply:
agree